### PR TITLE
Add folder size display on demand, via git/trees API. Fixes #19.

### LINKED ISSUE #19: Display folder sizes.
It would be great if it displays folder sizes also. Would be great for big projects as they have lots of folders/extensions and could easily track their sizes.

### PR DESCRIPTION
**Pull request by stevennoto**
Hi @harshjv, as I mentioned in #26, I wanted to revise this feature addition based on your feedback. I changed the implementation, so that now instead of fetching the full repo tree right away, it shows `---` for the size of folders, and if you click on that (or on the top repo size indicator), it will then fetch that folder size info and display it. It seems to work well for small repos and big ones too.

I have tested this and the code works well, although in its current form it is not super discoverable. I added `title` attributes to the clickable elements, but I was also considering using some sort of icon or link saying "Load Folder Sizes." I was curious what you thought about this before adding it though.

Let me know what you think, and thanks again for this useful extension!

### CODE REVIEW

**Comment by kryptokinght** [comment]
Umm, is this PR going to be merged anytime soon? We are waiting for the changes! :smile:

**Comment by stevennoto** [comment]
@kryptokinght, it depends on the maintainer. If you'd like the changes sooner, you can clone the repo and load the extension using this branch. See the readme for more info: https://github.com/harshjv/github-repo-size/#development

@harshjv, if you have a chance to review this, I can make any changes requested, and then you can merge it at your convenience. Thanks!

**Comment by harshjv** [comment]
Thank you @stevennoto @kryptokinght for your time and efforts. Sorry for the delay. I will review this today.

File: src/inject.js [hunk 1]
```diff
@@ -147,7 +173,16 @@ const checkForRepoPage = () => {
 
           const td = document.createElement('td')
           td.className = 'age'
-          td.innerHTML = '<span class="css-truncate css-truncate-target github-repo-size-td">' + getHumanReadableSize(t) + '</span>'
+          let label
+          if (t === 'dir') {
+            label = '---'
+            td.className += ' github-repo-size-folder'
+            td.title = "Click to load folder size"
```
Comment (harshjv) [inline src/inject.js]: Can you use single quotes here? This repo follows `js-standard` :)

**Comment by stevennoto** [comment]
Thanks for the review @harshjv, I'll look at these and try to get fixes in soon.

**Comment by mendel3** [comment]
@harshjv

**Pull request merged**